clustered/client/src/main/java/org/ehcache/clustered/client/internal/SimpleClusterTierManagerClientEntity.java

```diff
@@ -35,73 +32,56 @@
 import org.terracotta.exception.EntityException;
 
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /**
  * The client-side {@link Entity} through which clustered cache operations are performed.
  * An instance of this class is created by the {@link ClusterTierManagerClientEntityService}.
  * The server-side partner is the {@code EhcacheActiveEntity}.
  */
-public class SimpleClusterTierManagerClientEntity implements InternalClusterTierManagerClientEntity {
-
-  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleClusterTierManagerClientEntity.class);
+public class SimpleClusterTierManagerClientEntity implements ClusterTierManagerClientEntity {
 
   private final EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint;
   private final LifeCycleMessageFactory messageFactory;
 
-  private Timeouts timeouts = Timeouts.builder().build();
-
   public SimpleClusterTierManagerClientEntity(EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint) {
     this.endpoint = endpoint;
     this.messageFactory = new LifeCycleMessageFactory();
   }
 
-  @Override
-  public void setTimeouts(Timeouts timeouts) {
-    this.timeouts = timeouts;
-  }
-
   @Override
   public void close() {
     endpoint.close();
   }
 
   @Override
   public void validate(ServerSideConfiguration config) throws ClusterException, TimeoutException {
-    invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory.validateStoreManager(config), false);
+    invokeInternal(messageFactory.validateStoreManager(config), false);
```

Comment:
Is validate a lifecycle thing or a connection thing?  Should this be able to timeout?  If not we should get rid of the TimeoutException throw.

Reply:
For a manager, we decided that it should wait ever after and that the lease should take care of it. But indeed, it doesn't need the `TimeoutException` anymore